### DISCUSSION TITLE
life - 格莱姆的日常

### DISCUSSION THREAD

**Discussion by giscus** [tffats/shafish_blog#87] [Announcements]
# life - shafish.cn

Just write something about Work、Technology and Life.

https://shafish.cn/life/

**Comment by shafishcn** [comment]
有人猝不及防地走了

**Comment by shafishcn** [comment]
无人问津也好，技不如人也罢，你都要试着安静下来，去做自己该做的事情，而不是让烦恼和焦虑毁掉你本就不多的热情和定力，心可以碎，手不能停。

**Comment by shafishcn** [comment]
![rn_image_picker_lib_temp_15ef5355-133a-4498-b046-8e902ab74bda](https://user-images.githubusercontent.com/22794651/218526810-35fa43ab-f512-4aa1-b6f4-d7fda2e118cb.jpg)

**Reply by shafishcn** [reply]
可以看出：老夫是一个喜欢用铅笔的银

**Comment by shafishcn** [comment]
![rn_image_picker_lib_temp_b458bca9-e070-435f-9daf-5236044a97df](https://user-images.githubusercontent.com/22794651/218527104-cc4939eb-2156-40b4-a3e1-6c1e8dd3e3f1.jpg)

**Reply by shafishcn** [reply]
可以看出：老夫是一个爱健身的银

**Comment by shafishcn** [comment]
**，加班狗

**Comment by shafishcn** [comment]
今天去拔罐，是真的有点痛。运动、早睡，自律从今天做起，bro

**Comment by shafishcn** [comment]
打工好TM累人

**Comment by shafishcn** [comment]
有些梦 你明明拥有 却不为所动

有些梦 你不知不觉 已身在其中

《你好生活》

**Comment by jartins** [comment]
# **有点意思 哈哈哈哈**

**Reply by jartins** [reply]
欢迎来访[弹力笔记](https://elasticnotes.com "弹力笔记")

**Reply by shafishcn** [reply]
这阅读体验怎么感觉比我这好那么多 :heart_eyes:（虽然老夫的也很不错:blush:）

**Comment by jartins** [comment]
# 号外号外！ ssl证书过期了

**Reply by shafishcn** [reply]
嘿

**Comment by shafishcn** [comment]
睡前别喝水，特别是在周末。

**Comment by shafishcn** [comment]
10月11号，晚23℃多云，裸睡适宜，然加班

**Comment by shafishcn** [comment]
最近在折腾arch+i3wm，大学的时候买了一台二手的东芝笔记本用过一阵，现在打算装起来做主力机，不错，灰常不错。
![](https://picture.cdn.shafish.cn/blog/show.png)

**Comment by shafishcn** [comment]
13 4(语气助词1),23 3(语气助词2),日了狗了(语气助词3),晾个衣服还下雨淋湿
